app/renderer/components/main/main.js

```diff
@@ -252,12 +252,13 @@ class Main extends React.Component {
     }, { passive: true })
 
     ipc.on('scroll-touch-end', () => {
-      const distanceThreshold = getSetting(settings.SWIPE_NAV_DISTANCE)
+      const distanceThresholdX = getSetting(settings.SWIPE_NAV_DISTANCE)
+      const distanceThresholdY = 101
```

Comment:
Where does this 101 value come from? It might be a good idea to make it a constant, since here it appears to be a magic number

Reply:
it is the default value of `SWIPE_NAV_DISTANCE`, are you suggesting make it into settings constant? We don't allow users to config it so why bother?
https://github.com/brave/browser-laptop/pull/12434/files#diff-de5d4f1b019deecc5b1926970450546cR257 here doesn't appear to be magic number to you?

Reply:
ah ok- good to know that it's the default value 😄 👍